src/components/SaleCard/_saleCard.scss

```diff
@@ -67,9 +67,14 @@
   &__badge {
     border: 1px solid $badge-blue;
     color: $badge-blue;
+    font-size: 0.75rem;
```

Comment:
hmm shouldn't this be using one of our text tokens? this won't be responsive this way.